### PR TITLE
Add some obvious short options

### PR DESCRIPTION
**Pull request by rhatdan**
I chose to only add traditional ones like

--quiet, -q
and 
--user -u
And for some which could be specified multiple times I added 
-v, -e, -l

### CODE REVIEW

File: cmd/buildah/list.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 var (
 	listFlags = []cli.Flag{
 		cli.BoolFlag{
-			Name:  "quiet",
+			Name:  "quiet, q",
```
Comment (TomSweeneyRedHat) [inline cmd/buildah/list.go]: Thought we were doing 'omit' here to kill the column heading and 'quiet' to show just the id column.  fwiw, docker ps --quiet caused it to display on the numeric ids.

Comment (rhatdan) [inline cmd/buildah/list.go]: That sounds good to me.  Basically you want to add an option though correct, since there will still be a quiet option.  You just want to change the meaning.

Comment (TomSweeneyRedHat) [inline cmd/buildah/list.go]: Yes, I'd like to add an "omit" option here that would keep the table headers from printing when used.  "quiet" would be used to show just the image id's, much like docker does.  My main confusion here is I'd thought Nalin had already done this, but perhaps it wasn't merged?

**Comment by rhatdan** [comment]
@TomSweeneyRedHat I added a --noheading switch and changed quiet to work the way you wanted.

File: cmd/buildah/list.go (hunk 1 above)
Comment (nalind) [inline cmd/buildah/list.go]: In #32, yes.

Comment (rhatdan) [inline cmd/buildah/list.go]: Well lets do it here or in a separate pull request.

**Comment by nalind** [comment]
LGTM.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+  	e5fe19c
